test/e2e-tests.sh

```diff
@@ -43,7 +43,7 @@ failed=0
 
 # Run the integration tests
 header "Running Go e2e tests"
-go_test_e2e -timeout=35m ./test/... || failed=1
+go_test_e2e -timeout=35m -run TestVaultKMSSpire ./test/... || failed=1
```

Comment:
left over from debugging (I think this is from the other thing you were looking into?)

Reply:
yes. Just pushed the change to revert it back.